setup.py

```diff
@@ -12,7 +12,7 @@
     'url': 'http://github.com/PolySat/libproc',
     'author_email': 'cubesat@cubesat.org',
     'version': '0.1',
-    'install_requires': ['nose', 'tenjin', 'pyparsing'],
+    'install_requires': ['nose', 'tenjin', 'pyparsing<=2.4'],
```

Comment:
Do we want less than 2.4? Does it matter?

Reply:
I'm not sure it matters, it's obviously more inclusive to allow less than 2.4. I thought that was preferable to requiring 2.4 on the off chance 2.4 isn't available for some reason.